test/parallel/test-fs-watchfile.js

```diff
@@ -72,12 +72,15 @@ if (common.isLinux || common.isOSX || common.isWindows || common.isAix) {
     if (err) assert.fail(err);
 
     fs.watch(dir, common.mustCall(function(eventType, filename) {
+      clearInterval(interval);
       this._handle.close();
       assert.strictEqual(filename, 'foo.txt');
     }));
 
-    fs.writeFile(`${dir}/foo.txt`, 'foo', common.mustCall(function(err) {
-      if (err) assert.fail(err);
-    }));
+    const interval = setInterval(() => {
+      fs.writeFile(`${dir}/foo.txt`, 'foo', common.mustCall(function(err) {
+        if (err) assert.fail(err);
+      }));
+    }, 1);
```

Comment:
I figured it will not block the loop, but maybe stress the I/O. Although obviously the process shouldn't be doing anything else at this point.
I agree it's a least arbitrary value you could put here, so guess it could be read a sort of `nextTickInterval` idiom?